integration-tests/pom.xml

```diff
@@ -323,6 +323,23 @@
                 <module>google-cloud-functions-http</module>
                 <module>google-cloud-functions</module>
             </modules>
+            <build>
```

Comment:
There is already a build section further up.

Reply:
Oh, I missed it!